install_config/storage_examples/ceph_example.adoc

```diff
@@ -125,85 +139,77 @@ single user (non-shared storage).
 <4> This defines the volume type being used. In this case, the *rbd* plug-in is
 defined.
 <5> This is an array of Ceph monitor IP addresses and ports.
-<6> This is the Ceph secret, defined above. It is used to create a secure
+<6> `ceph-image` must be created on the Ceph cluster.
+<7> This is the Ceph secret, defined above. It is used to create a secure
```

Comment:
s/This is the Ceph secret, defined above./Enter the Ceph secret that you created.